src/libponyc/expr/call.c

```diff
@@ -569,21 +595,22 @@ static bool method_application(pass_opt_t* opt, ast_t* ast, bool partial)
     return false;
 
   AST_GET_CHILDREN(type, cap, typeparams, params, result);
+  bool bare = (ast_id(ast_child(type)) == TK_AT);
```

Comment:
In this context, `ast_child(type)` is the same as `cap` in the previous line, so you can use that variable.

Reply:
changed as suggested!